cmd/compose.go

```diff
@@ -188,6 +188,57 @@ func UploadPackageContents(r *util.Repo, appImage string, uploadPaths map[string
 	return newHashes, cmd.Wait()
 }
 
+func UploadPackageContentsToRemoteGuest(uploadPaths map[string]string, osvCmdline string, remoteGuestName string, verbose bool) error {
```

Comment:
This function is more or less a copy-paste from the original `UploadPackageContents` function only it doesn't start the instance beforehead. May I suggest to extract common logic into a function, say:

```go
func uploadFiles()
```

that both functions, `UploadPackageContents` and `UploadPackageContentsToRemoteGuest`, would use? The former would upload to `localhost`, the latter to the `remote`.